package.json

```diff
@@ -37,7 +37,7 @@
     ]
   },
   "peerDependencies": {
-    "@maplibre/maplibre-react-native": "^9.0.1",
+    "@maplibre/maplibre-react-native": ">=9.0.1 || >=10.0.0-alpha",
```

Comment:
peer dependency to any maplibre that is above 9.0.1, alphas will only match exact, so to use 10.0.0-alpha.X, the second part needs to be there.